content/chapter-03/ICanNotConnectLab.md

```diff
@@ -22,23 +22,23 @@
   * Download the application:
     * `curl -L -O https://raw.githubusercontent.com/vkhazin/cloudmigration-courseware/master/lab_support/count_lines.zip`
   * Unzip the app: `unzip count_lines.zip`
-  * This application will open the port 10000. Ensure you configure your network security correctly to make that port accessible
+  * This application will open the port 10000. Ensure you configure your network security correctly to make that port accessible. Try to find it your self, but the portal console *Security Groups* could be a good place to start
   * Change directory into the app folder to run it
```

Comment:
Try to think where "firewall" would be using the portal console, do you recall the discussion about *Security Groups* instead of firewalls on the public cloud?